.env.sepolia

```
@@ -1,46 +1,122 @@
+OP_NODE_L2_ENGINE_RPC=ws://execution:8551
 
-OP_NODE_BETA_EXTRA_NETWORKS=true
-OP_NODE_L2_ENGINE_AUTH=/tmp/engine-auth-jwt
-OP_NODE_L2_ENGINE_RPC=ws://geth:8551
-OP_NODE_LOG_LEVEL=info
-OP_NODE_METRICS_ADDR=0.0.0.0
-OP_NODE_METRICS_ENABLED=true
-OP_NODE_METRICS_PORT=7300
-OP_NODE_NETWORK=base-sepolia
+BASE_NODE_L2_ENGINE_RPC=http://execution:8551
+BASE_NODE_L2_ENGINE_AUTH=/tmp/engine-auth-jwt
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BASE_NODE_L2_ENGINE_AUTH key should go before the BASE_NODE_L2_ENGINE_RPC key

(UnorderedKey)